### PR TITLE
Wedding details update eb

### LINKED ISSUE #13: Create Photo List page
Include: 
- [x] Name of wedding 
- [x] date of wedding 
- [x] logo 
- [x] back button 
- [x] photo/family list

### LINKED ISSUE #30: When a user clicks on a `WeddingCard`
user should be able to see the individual wedding details

### LINKED ISSUE #42: Vendor should be able to email client to get photolist

### LINKED ISSUE #43: Conditional rendering to display photo list and guest list
- [ ] when a user clicks "view photo list" it should render the list over the photo section of the individaul wedding details page
- [ ] when a user clicks "edit list" it should render `GuestList` over the photo section of the individual wedding details page

### PR DESCRIPTION
**Pull request by ericberglund117**
# Pull Request

## What's this PR do? 

- [x] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [ ] Styling
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?

  * uses conditional rendering based on the state of the component to determine which button will be displayed to the user. 
  * Refactored the weddingDetails component to display the guest list, edit guest list, and allow the vendor to send a 'request info' email to the client 
  * Removes the requestPhotoListView state and button 

## Any background context you want to provide? 

  * these changes were decided upon as a group. 
  *  instead of rendering a second page on button clicks, there is conditional rendering for this one page
  * this will eliminate white space on the app pages.  

## What are the relevant tickets? 
* Addresses
* Closes #30 
* closes #43 
* closes #42 
* closes #13

### CODE REVIEW

File: src/components/WeddingDetails/WeddingDetails.tsx [hunk 1]
```diff
@@ -24,23 +27,82 @@ const WeddingDetails: React.FC<IndividualWedding> = ({
 	photoList
 }) => {
 
+		const [detailsView, setDetailsView] = useState(true)
+		const [photoListView, setPhotoListView] = useState(false)
+		const [editListView, setEditListView] = useState(false)
+		const emailBody = `Dear ${name},
+			it is time to fill out your family photo list! Please follow the link provided to complete the missing photo information. Feel free to reach out if you have any questions.
```
Comment (Kristiannmiller) [inline src/components/WeddingDetails/WeddingDetails.tsx:34]: We can definitely look into formatting this email a little better if time allows, but this works great for now

File: src/components/WeddingDetails/WeddingDetails.tsx [hunk 2]
```diff
@@ -24,23 +27,82 @@ const WeddingDetails: React.FC<IndividualWedding> = ({
...
 		<section className="detailsWrapper">
 			<div className="detailsHeader">
 				<h1 className="weddingTitle">{name} Wedding</h1>
 				<h2 className="weddingDate">{date}</h2>
 				<p className="weddingDetails">Email: {email}</p>
 				<p className="weddingDetails">Status: {familyPhotoList.length === 0 ? "Pending" : "Received"}</p>
+					{photoList.length === 0 &&
+					<StyledButton>
+						<div id="translate"></div>
+								<a className="link" id="requestListButton" href={`mailto:ericberglund117@gmail.com?subject=Family Photo List&body=${emailBody}`}>Request Photo List</a>
```
Comment (Kristiannmiller) [inline src/components/WeddingDetails/WeddingDetails.tsx:84]: Whoops! We left your email in there

**Comment by Kristiannmiller** [comment]
Looks great! Thanks for getting these views to work out! I think this style is going to translate a little easier to mobile

**Review by Kristiannmiller — APPROVED** [review]

File: src/components/WeddingDetails/WeddingDetails.tsx (hunk 2 above)
Comment (ericberglund117) [reply src/components/WeddingDetails/WeddingDetails.tsx:84]: Good catch, I GUESS we can replace that with the data from our server

File: src/components/WeddingDetails/WeddingDetails.tsx (hunk 1 above)
Comment (hannakim91) [reply src/components/WeddingDetails/WeddingDetails.tsx:34]: Agreed - nice to have in the future but at a lower priority than some other tasks

**Review by hannakim91 — APPROVED** [review]
Our `WeddingDetails` page is looking pretty functional! I think we're in good shape for the check-in later today.

**Pull request merged**